### ISSUE TITLE
Build issue with KDE Sdk 5.15

### ISSUE THREAD

**Issue by bellegarde-c** [fwbuilder/fwbuilder#110]
fwbuilder fails to build with last KDE/Qt stack.

```
/usr/include/QtCore/qtextstream.h:293:75: note: declared here
  293 | Q_CORE_EXPORT QT_DEPRECATED_VERSION_X(5, 15, "Use Qt::endl") QTextStream &endl(QTextStream &s);
      |                                                                           ^~~~
../src/iptlib/CompilerDriver_ipt_run.cpp:680:59: error: ‘QTextStream& QTextStreamFunctions::endl(QTextStream&)’ is deprecated: Use Qt::endl [-Werror=deprecated-declarations]
  680 |         if (have_ipv6) script << "  reset_iptables_v6" << endl;
      |                                                           ^~~~
In file included from /usr/include/QtCore/QTextStream:1,
                 from ../src/iptlib/../compiler_lib/CompilerDriver.h:38,
                 from ../src/iptlib/CompilerDriver_ipt.h:29,
                 from ../src/iptlib/CompilerDriver_ipt_run.cpp:43:
/usr/include/QtCore/qtextstream.h:293:75: note: declared here
  293 | Q_CORE_EXPORT QT_DEPRECATED_VERSION_X(5, 15, "Use Qt::endl") QTextStream &endl(QTextStream &s);
      |                                                                           ^~~~
cc1plus: all warnings being treated as errors
[143/1258] Building CXX object src/cisco_li...les/fwbcisco.dir/PolicyCompiler_cisco.cpp.o
ninja: build stopped: subcommand failed.

```

**Comment by nedrichards** [comment]
Note - this release upgrades to GCC 10 as the compiler.

**Comment by sirius** [comment]
I've fixed Qt 5.15 compatability on current master (17b6279e22037a351be731c2059548b120df93da)
